guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -0,0 +1,1121 @@
+{
```

Comment:
<p>Since deep-learning-essentials conda package is no longer shipped, please update the install instruction. You can perhaps list a long conda install command or split that into two.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>